src/features/hotspots/settings/discovery/filterDiscoveryResponses.ts

```diff
@@ -0,0 +1,15 @@
+import { DiscoveryResponse } from '../../../../store/discovery/discoveryTypes'
+
+export default (hotspotAddress: string, responses: DiscoveryResponse[]) => {
+  const filtered = responses.filter((response, index, allResponses) => {
```

Comment:
Hotspots can respond with "self" and there can be duplicates. Need to filter through them. Wallet API will prevent self from coming through in the future.